runtime/tests/org.eclipse.core.tests.runtime/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Eclipse Core Tests Runtime
 Bundle-SymbolicName: org.eclipse.core.tests.runtime; singleton:=true
-Bundle-Version: 3.21.500.qualifier
+Bundle-Version: 3.21.400.qualifier
```

Comment:
Decreasing version looks totally wrong!

Reply:
If the version bump made it to an I-build please don't revert at all as people that updated will never get changes after the revert.

Reply:
Yes. Since the commit has been merged 1 hour ago, it is not part of an I-Build yet, so I think reverting is okay and  the right thing to do?

Reply:
Yes